plugins/woocommerce/src/Admin/Features/PaymentGatewaySuggestions/PaymentGatewaySuggestionsDataSourcePoller.php

```diff
@@ -33,7 +33,20 @@ class PaymentGatewaySuggestionsDataSourcePoller extends DataSourcePoller {
 	 */
 	public static function get_instance() {
 		if ( ! self::$instance ) {
-			self::$instance = new self( self::ID, self::DATA_SOURCES );
+			// Add country query param to data sources.
+			$base_location = wc_get_base_location();
```

Comment:
I'm curious if we could use `get_option( 'woocommerce_default_country', 'US:CA' )` instead? Or, do you know if `wc_get_base_location` is more accurate/reliable?

Reply:
I use `wc_get_base_location` because we only need "country". We don't need "state/region". 😄

Reply:
is same thing 😆
https://github.com/woocommerce/woocommerce/blob/e18a531e85b2a8c5e667526844f14d388b0c649a/plugins/woocommerce/includes/wc-core-functions.php#L1318